README.md

```diff
@@ -1,50 +1,43 @@
 # Universal Permissionless Rewards Distributor
 
-A universal permissionless rewards distributor written in Solidity. It allows the distribution of any reward token (different reward tokens are possible simultaneously) based on a Merkle tree distribution, and using ERC20 allowance of a treasury.
+A universal permissionless rewards distributor written in Solidity.
+It allows the distribution of any reward token (different reward tokens are possible simultaneously) based on a Merkle tree distribution, and using ERC20 allowance of a treasury.
 
-The singleton contract allows any treasury to distribute rewards to any address, based on a Merkle tree. The Merkle root is stored in the contract. The Merkle root can be updated by whitelisted users. The distribution owner can freeze, force update, or suggest a new treasury. The treasury must accept the role and set an allowance to distribute rewards.
+The singleton contract allows any treasury to distribute rewards to any address, based on a Merkle tree.
```

Comment:
but it doesn't use an allowance-based mechanism anymore right?
and there's no treasury anymore

Reply:
Yes there are no treasury

Reply:
Yes this is added in point 2 of https://github.com/morpho-org/universal-rewards-distributor/issues/78